src/main/java/service/StrategyProcessor.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface StrategyProcessor {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```